### DISCUSSION TITLE
Exception: Scope not found

### DISCUSSION THREAD

**Discussion by R0N7N** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#79] [Q&A] [answered]
Hi, I set up the config as stated in the docs.
But I am getting a "Exception: Scope not found" Error when I try to run the script.

Any tips on what to check?
I filled out all the required fields.

Greetings

**Comment by chriskyfung** [comment, marked as answer]
Please authorize with your Google account to gain the permissions.

**Reply by chriskyfung** [reply]
Learn more: https://developers.google.com/apps-script/guides/client-verification
